lib/matplotlib/animation.py

```diff
@@ -758,55 +869,72 @@ def _stop(self, *args):
     def save(self, filename, writer=None, fps=None, dpi=None, codec=None,
              bitrate=None, extra_args=None, metadata=None, extra_anim=None,
              savefig_kwargs=None):
-        '''
-        Saves a movie file by drawing every frame.
-
-        *filename* is the output filename, e.g., :file:`mymovie.mp4`
-
-        *writer* is either an instance of :class:`MovieWriter` or a string
-        key that identifies a class to use, such as 'ffmpeg' or 'mencoder'.
-        If nothing is passed, the value of the rcparam `animation.writer` is
-        used.
-
-        *dpi* controls the dots per inch for the movie frames. This combined
-        with the figure's size in inches controls the size of the movie.
-
-        *savefig_kwargs* is a dictionary containing keyword arguments to be
-        passed on to the 'savefig' command which is called repeatedly to save
-        the individual frames. This can be used to set tight bounding boxes,
-        for example.
-
-        *extra_anim* is a list of additional `Animation` objects that should
-        be included in the saved movie file. These need to be from the same
-        `matplotlib.Figure` instance. Also, animation frames will just be
-        simply combined, so there should be a 1:1 correspondence between
-        the frames from the different animations.
-
-        These remaining arguments are used to construct a :class:`MovieWriter`
-        instance when necessary and are only considered valid if *writer* is
-        not a :class:`MovieWriter` instance.
-
-        *fps* is the frames per second in the movie. Defaults to None,
-        which will use the animation's specified interval to set the frames
-        per second.
-
-        *codec* is the video codec to be used. Not all codecs are supported
-        by a given :class:`MovieWriter`. If none is given, this defaults to the
-        value specified by the rcparam `animation.codec`.
-
-        *bitrate* specifies the amount of bits used per second in the
-        compressed movie, in kilobits per second. A higher number means a
-        higher quality movie, but at the cost of increased file size. If no
-        value is given, this defaults to the value given by the rcparam
-        `animation.bitrate`.
-
-        *extra_args* is a list of extra string arguments to be passed to the
-        underlying movie utility. The default is None, which passes the
-        additional arguments in the 'animation.extra_args' rcParam.
-
-        *metadata* is a dictionary of keys and values for metadata to include
-        in the output file. Some keys that may be of use include:
-        title, artist, genre, subject, copyright, srcform, comment.
+        '''Saves a movie file by drawing every frame.
+
+        Parameters
+        ----------
+
+        filename : str
+            the output filename, e.g., :file:`mymovie.mp4`
```

Comment:
It's odd that some of these start with a lower-case letter, but then are followed by another capitalized sentence.